### ISSUE TITLE
Error and build exception on Nativescript 1.7

### ISSUE THREAD

**Issue by calebeaires** [EddyVerbruggen/nativescript-local-notifications#5]
Build failed with an exception

```
:cleanLocalAarFiles
:ensureMetadataOutDir
:collectAllJars
:buildMetadata
Exception in thread "main" java.lang.IllegalArgumentException: Class com.google.android.gms.ads.formats.NativeAd$Image conflict: /Users/macbook/apptest/platforms/android/build/intermediates/exploded-aar/com.google.android.gms/play-services-ads/7.5.0/jars/classes.jar and /Users/macbook/apptest/node_modules/nativescript-local-notifications/node_modules/nativescript-plugin-google-play-services/platforms/android/google-play-services_lib/libs/google-play-services.jar
	at com.telerik.metadata.ClassRepo.cacheJarFile(ClassRepo.java:21)
	at com.telerik.metadata.Builder.build(Builder.java:38)
	at com.telerik.metadata.Generator.main(Generator.java:44)
:buildMetadata FAILED

* What went wrong:
Execution failed for task ':buildMetadata'.
> Process 'command '/Library/Java/JavaVirtualMachines/jdk1.7.0_79.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1
```

Using:
Nativescript 1.7.1
Android 5.1

Extra modules:
nativescript-admob
lodash

**Comment by MiroValchev** [comment]
Hi @EddyVerbruggen and thanks for the great plugin!

However, after adding it, build for android fails with the following message.

Exception in thread "main" java.lang.IllegalArgumentException: Class com.telerik.widget.chart.R$integer conflict: \platforms\android\build\intermediates\classes\nativescript-telerik-ui\debug and \platforms\android\build\intermediates\classes\nativescript-local-notificationsNativescript-plugin-google-play-servicesNativescript-telerik-ui\debug
        at com.telerik.metadata.ClassRepo.cacheJarFile(ClassRepo.java:21)
        at com.telerik.metadata.Builder.build(Builder.java:41)
        at com.telerik.metadata.Generator.main(Generator.java:44)
:buildMetadata FAILED

In my package.json

"dependencies": {
    "nativescript-local-notifications": "^1.0.1",
    "nativescript-sqlite": "^1.0.8",
    "nativescript-telerik-ui-pro": "^1.0.1",
    "tns-core-modules": "1.7.1"
  }

**Comment by EddyVerbruggen** [comment]
Looks like dependencies with other plugins fail because of version conflicts in dependencies these plugins have. Let me try to use gradle for this plugin as well soon, like I did for [AdMob](https://github.com/EddyVerbruggen/nativescript-admob/tree/master/platforms/android)

**Comment by MiroValchev** [comment]
Sure, let me know if I can help somehow.

**Comment by EddyVerbruggen** [comment]
I __think__ this was releated to the length of the cloned project folder name. When creating a new NativeScript app with the name `nativescript-local-notifications-demo` the CLI will not allow it. But when cloning the repo you won't get that warning. So I've explicitly mentioned that in the readme of the demo app.

What also might have been the case is that the Google Play Services library was causing problems. I've just released 1.0.2 to npm which should no longer bother you.

Please test with a fresh CLI project to verify these changes.

**Comment by MiroValchev** [comment]
Works like a charm :)
